src/FieldWarning/Assets/Camera/SlidingCameraBehaviour.cs

```diff
@@ -200,8 +209,64 @@ private void TiltCameraIfNearGround(float oldAltitude)
     private void ClampCameraAltitude()
     {
         _targetPosition.y = Mathf.Clamp(
-                _targetPosition.y, 
+                _targetPosition.y,
                 Terrain.activeTerrain.SampleHeight(_targetPosition) + _minAltitude,
                 _maxAltitude);
     }
+
+    private void PanFromScreenBorder()
+    {
```

Comment:
This can be cleaned up a lot, at the very least by adding some strategic empty lines. I won't delay the merge because of it, but its something to keep in mind.